unix/x0vncserver/x0vncserver.cxx

```diff
@@ -281,11 +281,6 @@ int main(int argc, char** argv)
 
   Configuration::enableServerParams();
 
-  // FIXME: We don't support clipboard yet
-  Configuration::removeParam("AcceptCutText");
-  Configuration::removeParam("SendCutText");
-  Configuration::removeParam("MaxCutText");
```

Comment:
Could you update the man page as well now that these are available?

Reply:
Done.